src/config/index.js

```diff
@@ -5,7 +5,6 @@ module.exports = {
   },
   database: {
     uri: process.env.NODE_ENV === 'production'
-      ? process.env.MONGO_URI
-      : 'mongodb://localhost:27017/codinggardencommunity',
+      ? process.env.MONGO_URI : 'mongodb://spiray:abc123abc123@ds139946.mlab.com:39946/codinggardencommunity',
```

Comment:
Do not include a real mongo address. Anyone can now connect to that DB.

Instead, you can update uri to be this:

```js
uri: process.env.MONGO_URI || 'mongodb://localhost:27017/codinggardencommunity'
```

And then add your mongo connection url in the .env file.

Reply:
Thanks, Done!